pkg/ride/functions_proto.go

```diff
@@ -310,6 +310,23 @@ func intFromState(env Environment, args ...rideType) (rideType, error) {
 	return rideInt(entry.Value), nil
 }
 
+func intFromSelfState(env Environment, args ...rideType) (rideType, error) {
+	k, err := extractKey(args)
+	if err != nil {
+		return rideUnit{}, nil
```

Comment:
return err or errors.Wrap(err, ...

Reply:
No, it's a special class of functions that swallows all errors.